docs/plugins/symbolic_explanations.rst

```diff
@@ -0,0 +1,17 @@
+Symbolic Explanations
+====================
+
+This plugin is capable of answering similar questions as the Partial Dependencies plugin, i.e.:
```

Comment:
I personally think an explanation of what symbolic explanations are in the first sentence would be best; so just adding one more sentence before this one.

Reply:
Added a sentence in the beginning.